### DISCUSSION TITLE
Submit an idea

### DISCUSSION THREAD

**Discussion by Willyfrog** [mattermost/community#123] [Practical Ideas]
Have a button that sends the user idea to Mattermost without leaving Mattermost. Ideally it could be published here, but I think the user would need a GitHub account, so we might need to send it somewhere else before publishing into a discussion

**Comment by esethna** [comment]
Got a ticket for that! https://mattermost.atlassian.net/browse/MM-29724

It would utilize the NPS survey infrastructure but be user initiated

**Reply by esethna** [reply]
Caveat here is that it's not public, it just goes to our Looker dashboards and internal integrations
